util/topgen/lib.py

```diff
@@ -495,3 +495,15 @@ def make_bit_concatenation(sig_name: str,
         acc += [',', item_indent, item]
     acc += ['\n', ' ' * end_indent, '}']
     return ''.join(acc)
+
+
+def param_value (ip_param, module_params_decl):
+    '''Return param value of an exposed parameter.
+       If there are no explicitly declared value under module, use default.
```

Comment:
nit: value -> value**s**

Reply:
Actually, I think it should probably be "If there is no explicitly..."? Also, could you add a newline before the second sentence? The usual style for Python docstrings is like Git commit messages: a single line title, then zero or more paragraphs separated by blank lines.

Reply:
ah yes, the next line seems to be singular and this line plural for some reason..let me switch to singular for both.